Add changeset publish flow update (#105)
pnpm/pnpm.github.io: docs/using-changesets.md

## Releasing changes

1. Run `pnpx changeset version`. This will bump the versions of the packages
   previously specified with `pnpx changeset` (and any dependents of those) and
   update the changelog files.
2. Run `pnpm install`. This will update the lockfile and rebuild packages.
3. Commit the changes.
4. Run `pnpm publish -r`. This command will publish all packages that have
   bumped versions not yet present in the registry.

## Using GitHub Actions

To automate the process, you can use `changeset version` with GitHub actions.

### Bumb up packages

> The action will detect when changesets files arrives in the `main` branch, the action will open a new PR listing all the packages will be bump up. Once is merged the packages will be updated and you can decide whether publising or not by adding the `publish` property.

Reviewer: ```suggestion
The action will detect when changeset files arrive in the `main` branch, the action will open a new PR listing all the packages with bumped versions. Once merged, the packages will be updated and you can decide whether to publish or not by adding the `publish` property.
```

### Publishing

By adding `publish: pnpm ci:publish` which is a script that executes `changeset publish`

> will publish on the registry once the PR opened by `changeset version`.

Reviewer: ```suggestion
will publish to the registry once the PR is opened by `changeset version`.
```
